src/content/changes/highlights/2021-06-11.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2021-06-14T10:00:00
+title: Highlights for the week ending Jun 11, 2021
+changes_categories:
+- Highlights
+owner:
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+As we aim to align our terminology with the Kubernetes project where possible, we started replacing the term _master node_ with _control plane node_.
+
+## Managed apps
+
+[Loki v0.2.0](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.2.0/) comes with upstream version 2.2.1. Please note that there is a manual intervention required when upgrading to this version. Check the release info for details.
```

Comment:
```suggestion
[Loki v0.2.0](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.2.0/) comes with upstream version 2.2.1. You can now use an existing secret (such as sealed secrets) for configuration, making the app more flexible. Please note that there is a manual intervention required when upgrading to this version. Check the release info for details.
```

Reply:
@ubergesundheit I suggested something. Does it make sense?

(All I really know is this was a blocker for a customer to use our Loki set up)

Reply:
Perfect! <3